### PR TITLE
Ek/score table

### PR DESCRIPTION
**Pull request by Erveftick**
- Added scoreTable
- Refactored header: added antd menu, replaced bootstrap selectors to antd
- Added scoretable route on front
- Minor improvements

### CODE REVIEW

File: api/api.lesson.js [hunk 1]
```diff
@@ -27,6 +27,11 @@ class LessonAPI {
     const { data } = await instanceAxios.post(`/lesson/mark/${id}`, { mark, lessonStepId })
     return data
   }
+
```
Comment (Algoritm211) [inline api/api.lesson.js]: Scoretable more related to the course, not lesson. Isn't it?

File: components/CoursePage/CoursePage.jsx [hunk 2]
```diff
@@ -1,128 +1,153 @@
...
-                  <button className="search-btn">
-                    <i className="fas fa-search grid-100"/>
-                  </button>
-                </p>
-              </form>
-            </section>
+          <div className="container pb-3">
+            <div className="d-flex align-items-center pb-3">
+              <Select
+                defaultValue={t("category")}
```
Comment (Algoritm211) [inline components/CoursePage/CoursePage.jsx:73]: Some useless spacings

Снимок экрана 2022-04-03 в 13 23 35

File: components/MainPage/MainPage.jsx [hunk 3]
```diff
@@ -44,96 +48,148 @@ const MainPage = ({ size }) => {
...
-              <a style={{ textDecoration: 'none' }}>
-                <div className="carousel-card" style={{ backgroundColor: '#B2CCFC' }}>
-                  <h3 className="carousel-title" style={{ color: '#466aa8' }}>{t('it')}</h3>
-                  <p className="carousel-subtitle" style={{ color: '#466aa8' }}>7 {t('existCourses')}</p>
+              </a>
+            </Link>
+            <Link href={"/"}>
+              <a style={{ textDecoration: "none" }}>
+                <div
+                  className="carousel-card"
```
Comment (Algoritm211) [inline components/MainPage/MainPage.jsx:176]: ditto

Снимок экрана 2022-04-03 в 13 25 04

File: components/Navbars/CourseNavbar.jsx [hunk 4]
```diff
@@ -21,6 +21,55 @@ const CourseNavbar = () => {
...
+      textLink: t("teaching"),
+    },
+    {
+      routePath: "/settings",
+      iconClass: "fas fa-cog",
+      textLink: t("settings"),
+    },
+  ];
+
+  const navbarMenu = profileNavbarData.map(
```
Comment (Algoritm211) [inline components/Navbars/CourseNavbar.jsx]: Looks like it should be separate component with props

Снимок экрана 2022-04-03 в 13 25 44

File: components/Navbars/ProfileNavbar.jsx [hunk 5]
```diff
@@ -1,55 +1,79 @@
-import React, { useState } from 'react'
-import Link from 'next/link'
-import { useRouter } from 'next/router'
-import CreateCourseModal from '../EditorPage/CreateCourseModal/CreateCourseModal'
-import { useTranslation } from 'next-i18next'
+import React, { useState } from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
```
Comment (Algoritm211) [inline components/Navbars/ProfileNavbar.jsx]: Check your prettier work, we use single quotes in out files

File: styles/profileMain.css [hunk 6]
```diff
@@ -1,322 +1,333 @@
...
-        font-size: 12px;
-    }
-}
-@media (max-width: 575px) {
-    .profile-mobile{
-        display: flex;
-        justify-content: center;
-    }
+
+.ant-menu-horizontal:not(.ant-menu-dark) > .ant-menu-item:hover::after, .ant-menu-horizontal:not(.ant-menu-dark) > .ant-menu-submenu:hover::after, .ant-menu-horizontal:not(.ant-menu-dark) > .ant-menu-item-active::after, .ant-menu-horizontal:not(.ant-menu-dark) > .ant-menu-submenu-active::after, .ant-menu-horizontal:not(.ant-menu-dark) > .ant-menu-item-open::after, .ant-menu-horizontal:not(.ant-menu-dark) > .ant-menu-submenu-open::after, .ant-menu-horizontal:not(.ant-menu-dark) > .ant-menu-item-selected::after, .ant-menu-horizontal:not(.ant-menu-dark) > .ant-menu-submenu-selected::after{
```
Comment (Algoritm211) [inline styles/profileMain.css:327]: Maybe create some separate file for this utils?

File: pages/scoretable/[id].js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+import Header from "../../components/Header/Header";
+import CourseAPI from "../../api/api.course";
+import withEditProtect from "../../components/HOC/withEditProtect";
+import { wrapper } from "../../store/store";
+import ScoreTable from "../../components/ScoreTable/ScoreTable";
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+
+const MarksTable = ({ marks }) => {
+  return (
+    <div>
```
Comment (Algoritm211) [inline pages/scoretable/[id].js]: Can we here use React.Fragment instead of div?

File: api/api.lesson.js (hunk 1 above)
Comment (Erveftick) [reply api/api.lesson.js]: It's a legacy code, idk why it still here. I'll remove it

File: components/CoursePage/CoursePage.jsx (hunk 2 above)
Comment (Erveftick) [reply components/CoursePage/CoursePage.jsx:73]: Strange, 'cause even it github i can't see any spacing

File: components/MainPage/MainPage.jsx (hunk 3 above)
Comment (Erveftick) [reply components/MainPage/MainPage.jsx:176]: the same, i can't see any spacing

**Review by Algoritm211 — APPROVED** [review]

**Pull request merged**